### DISCUSSION TITLE
Cant enable "people" in settings

### DISCUSSION THREAD

**Discussion by kuduacz** [photoprism/photoprism#3921] [Q&A] [answered]
---------------------------------------------------------------------------

#### 1. What is not working as documented?

Cant enable "people" in settings. when click on it it marked and after moment disable by himself.

Using photoprism on docker.

in config.yaml its on "true"

Build [231021-9abea5b55](https://docs.photoprism.app/release-notes/)

AMD64 arch N1000 processor 16 gb ram

SSD

**Comment by lastzero** [comment]
I'll convert this to a discussion as it appears to be a local configuration issue or a misunderstanding of how the software works.

**Comment by graciousgrey** [comment, marked as answer]
Do you have tensorflow enabled? This is required for the facial recognition.

**Reply by kuduacz** [reply]
Solved. dont know about that. Ty and sorry.

**Reply by graciousgrey** [reply]
You are welcome :)
